### DISCUSSION TITLE
Convert Image data to video

### DISCUSSION THREAD

**Discussion by ClosetGeek-Git** [microsoft/AirSim#4665] [General]
Hi, this might be a weird request, but to most accurately simulate a real pipeline I think it would be beneficial to be able to convert image data into video, and then users can process this video using an array of available tools such as TensorFlow. H264 encoding is fairly standard but mjpeg would probably be more simple, but I wouldn't know I have no experience with video encoding.

**Comment by ClosetGeek-Git** [comment]
This is enough to get the imagination going https://stackoverflow.com/questions/1595945/creating-my-own-mjpeg-stream, but not a solution.

**Comment by ClosetGeek-Git** [comment]
Is this forum dead?

**Reply by xxEoD2242** [reply]
Hey @ClosetMonkey, AirSim is being archived coming soon. We are continuing the open source project [here](https://github.com/CodexLabsLLC/Colosseum) with a new version called Colosseum.

You can create a new discussion there as well. As for your actual question, how familiar are you with ffmpeg?

**Reply by ClosetGeek-Git** [reply]
Not very. I used ffmpeg for a WordPress module around 15 years ago to converted the format of uploaded videos, but I've never used it with media streams or image feeds.
